Makefile

```diff
@@ -165,6 +165,7 @@ bundle-push: ## Push the bundle image.
 catalog: opm ## Generate catalog manifests and then validate generated files.
 	$(OPM) render --output=yaml $(BUNDLE_IMG) $(OPM_RENDER_OPTS) > catalog/odf.yaml
 	$(OPM) render --output=yaml $(OCS_BUNDLE_IMG) $(OPM_RENDER_OPTS) > catalog/ocs.yaml
+	$(OPM) render --output=yaml $(OCS_CLIENT_BUNDLE_IMG) $(OPM_RENDER_OPTS) > catalog/ocs-client.yaml
```

Comment:
please add catalog/ocs-client.yaml to .gitignore file as well

Reply:
done.